src/internal/m365/collection/groups/backup.go

```diff
@@ -153,11 +179,13 @@ func populateCollections(
 		// }
 
 		// ictx = clues.Add(ictx, "previous_path", prevPath)
-
+		// TODO: Neha check this
+		var fields []string
 		// TODO: the handler should provide this implementation.
+		// TODO: if we doing this messages are items for us.
```

Comment:
I'm not sure what this comment is supposed to mean.  Can you please explain?

Reply:
I was figuring out some changes. This comment is no more needed.